index.js

```diff
@@ -18,12 +22,29 @@ function onClosed() {
 	mainWindow = null;
 }
 
+function overrideRefreshWindowShotcuts(){
+	/*const ret = */globalShortcut.register('CmdOrCtrl+R', () => {
+		//console.log('CmdOrCtrl+R')
```

Comment:
Is this basically saying on refresh do nothing? If so perhaps a comment would be beneficial?

Reply:
Yes, It is basically  doing that . I will leave a comment on it saying what it does.